experiments/experiment.py

```diff
@@ -48,43 +57,58 @@ def run(self, session, desc="", save=True, force=False, parse=True):
         if os.path.exists(self.result_file) and not force:
             print("Reading report (%s)" % self.report_file)
             self.read_report()
-            
+
         print("-"*30)
-        print("Seed:     %d" % self.seed)
-        print("Model:    %s" % self.model_file)
-        print("Task:     %s" % self.task_file)
-        print("Result:   %s" % self.result_file)
-        print("Report:   %s" % self.report_file)
+        print("Seed:     {}".format(self.seed))
+        print("Model:    {}".format(self.model_file))
+        print("Task:     {}".format(self.task_file))
+        print("Result:   {}".format(self.result_file))
+        print("Report:   {}".format(self.report_file))
         n = self.n_session * self.n_block * self.n_trial
-        print("Sessions: %d (%d trials)" % (self.n_session, n))
+        print("Sessions: {} ({} trials)".format(self.n_session, n))
         print("-"*30)
-                
+
         if not os.path.exists(self.result_file) or force:
             index = 0
             records = np.zeros((self.n_session, self.n_block, self.n_trial),
                                dtype=self.task.records.dtype)
-            pool = Pool(4)
-            for result in tqdm(pool.imap_unordered(session, [self,]*self.n_session),
+
+            pool = Pool() # the number of process is set by multiprocessing.cpu_count()
```

Comment:
Ok, it would be better to make it explicit just in case the Pool defaults change at some point